juriscraper/opinions/united_states/state/ill.py

```diff
@@ -79,11 +92,19 @@ def _get_precedential_statuses(self) -> List[str]:
 
     def _get_docket_numbers(self) -> List[str]:
         """Get the docket number from citation.
-
+        Examples:
+            Citation         - Docket
+            "2019 IL 123318" - 123318
+            "2020 IL 124831" - 124831
+            "2021 IL 126432" - 126432
         Returns: List of docket numbers
         """
         dockets_numbers = []
         for case in self.cases:
-            m = re.search(self.docket_re, case["neutral_citation"])
-            dockets_numbers.append(m.group("docket"))
+            match = re.search(self.docket_re, case["neutral_citation"])
+            if match:
+                dockets_numbers.append(match.group("docket"))
+            else:
+                logger.critical(f"Could not find docket for case: '{case}'")
+                continue
         return dockets_numbers
```

Comment:
Won't this method crash the entire scraper if a docket number is missing in a case?   Thats a problem... I think.

Reply:
Should I add back the `InsanityException` exception for this case @flooie ?

Reply:
No.

Reply:
Use the pattern I gave you on slack.  It should be able to guide you.  Insanity executions are a relic before we started scraping linearly.  They dont server a purpose now.

Reply:
Done